.gitignore

```diff
@@ -17,6 +17,7 @@ msbuild.log
 # please do not commit any app.config files.
 app.config
 !src/ResourceManager/Profile/Commands.Profile.Test/App.config
+!src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config
```

Comment:
@rahuldutta90 I don't believe the changes to the `App.config` and `Newtonsoft.Json` that are being made in this PR are necessary -- you should be able to update the version of your SDK without having to touch the version of `Newtonsoft.Json`. Looking at the package, it still depends on 6.0.8 (or higher) for desktop, so you shouldn't need to update the `.csproj` and `.config` files outside of the SDK version itself.

Reply:
@cormacpayne I pinged Mark about this. "I have been trying to run live unit tests for datalake powershell after almost 3 months but its failing with "Could not load file or assembly 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. " All the azure datalake sdks depend on 6.0.8 version of newtonsoft. Does any common powershell packages change to depend on 9.0.0?"

It does not allow me to run live unit tests unless I do this.

Reply:
@cormacpayne I checked the module, it is working fine. I suspect its the tests only.